code/modules/urist/mob/spoopymobs.dm

```diff
@@ -702,4 +702,65 @@
 /mob/living/simple_animal/hostile/urist/stalker/UnarmedAttack(var/atom/A, var/proximity)
 	..()
 	if(!client && caution) //run awaaaay!
-		HuntingTeleport()
\ No newline at end of file
+		HuntingTeleport()
+
+//Angels, old testament style
+
+/mob/living/simple_animal/hostile/urist/angel
+	name = "\improper angel"
+	desc = "A suposedly divine being known to bring the wrath of their diety."
+	response_help = "tries to poke"
+	response_disarm = "tries to shove"
+	response_harm = "tries to hit"
+	attacktext = "smites down"
+	icon= 'icons/uristmob/simpleanimals.dmi'
+	icon_state = "bluespace-angel"
+	icon_living = "bluespace-angel"
+	icon_dead = ""
+	casingtype = /obj/effect/decal/cleanable/ash
+	faction = "divine"
+	maxHealth = 100
+	health = 100
+	projectiletype = /obj/item/projectile/energy/holy
+	projectilesound = 'sound/magic/fireball.ogg'
+	attack_sound = 'sound/weapons/slash.ogg'
+	ranged = 1//shoots and then charges.
+	harm_intent_damage = 0
+	melee_damage_lower = 10
+	melee_damage_upper = 20
+
+/mob/living/simple_animal/hostile/urist/angel/death()
+	..()
+	visible_message("<span class='danger'>The [src.name] wails and disappears!</span>")
+	playsound(src.loc, 'sound/effects/angel-reveal.ogg', 50, 1)
+	flick("forgotten_die", src)
+	sleep(4)
+	qdel(src)
+	return
+
+/obj/item/projectile/energy/holy
+	name = "holy fire"
+//	icon = 'icons/obj/projectiles.dmi'
```

Comment:
can you move these sprites to icons\urist\items\effects.dmi? overwriting Bay icon files is something we try to avoid.

Reply:
copy that. closing this RP to pack it with the big planet one